tahowallet/extension
# Add private key onboarding flow

Resolves https://github.com/tahowallet/extension/issues/3069

### What

This PR adds private key flow to tabbed onboarding and allows wallet to show these wallets in the UI correctly

- make seed phrase safe input a separate, shared component
- refactor seed phrase form to make it reusable
- add new page for private key import
- show private key imported wallets in a new section on the accounts list

https://user-images.githubusercontent.com/20949277/223729085-1c5c2c18-3f4d-4fb9-8dbc-0193c458f902.mov

![image](https://user-images.githubusercontent.com/20949277/223158147-0d94c618-0a45-48cd-b042-8faf1e27387a.png)

### Testing

1. Go through the private key onboarding flow
2. Import account with private key
3. Check how it behaves in the wallet
4. Import other accounts, check wallets list - private key wallets should be in `Generated and imported` section

Latest build: [extension-builds-3119](https://github.com/tahowallet/extension/suites/12013923000/artifacts/631100120) (as of Tue, 04 Apr 2023 09:23:44 GMT).
